rootfs/scheduler/__init__.py

```diff
@@ -604,13 +604,15 @@ def _create_rc(self, name, image, command, **kwargs):
 
         # Check if it is a slug builder image.
         # Example format: golden-earrings:git-5450cbcdaaf9afe6fadd219c94ac9c449bd62413s
-        vcs, sha = image[image.index(':')+1:].split('-')
-        if vcs == 'git' and len(sha) in [8, 40] and app_type == 'web':
-            imgurl = 'http://{}/git/home/{}/push/slug.tgz'.format(
-                settings.S3EP,
-                image.replace(':', '-')
-            )
-            TEMPLATE = RCB_TEMPLATE
+        image_name = image[image.index(':')+1:]
+        if '-' in image_name:
+            vcs, sha = image_name.split('-')
+            if vcs == 'git' and len(sha) in [8, 40] and app_type == 'web':
+                imgurl = 'http://{}/git/home/{}/push/slug.tgz'.format(
+                    settings.S3EP,
+                    image.replace(':', '-')
```

Comment:
is this `image` variable coming from above? want to make sure this isn't supposed to be `image_name`